tools/c7n_azure/c7n_azure/actions/delete.py

```diff
@@ -9,24 +9,27 @@ class DeleteAction(AzureBaseAction):
     ARM resource type supported by Cloud Custodian.
 
     :example:
+
     This policy will delete any ARM resource with 'test' in the name
 
     .. code-block:: yaml
 
-            policies:
-              - name: delete-test-resources
-                resource: azure.armresource
-                description: |
-                  Deletes any ARM resource with 'test' in the name
-                filters:
-                  - type: value
-                    key: name
-                    value: test
-                    op: contains
-                actions:
-                 - type: delete
+        policies:
```

Comment:
Where should the `p` in `policies` be aligned here in the ideal case?

Reply:
In most cases it is aligned to `o` letter in `code-`.. I'm not sure if we can enforce this check atm